apps/supernova/src/components/silences/SilencesList.jsx

```diff
@@ -38,11 +40,46 @@ my-px
 const SilencesList = () => {
   const silences = useSilencesItems()
   const [visibleSilences, setVisibleSilences] = useState(silences)
-  const { setSilencesStatus, setSilencesRegEx } = useSilencesActions()
   const status = useSilencesStatus()
   const regEx = useSilencesRegEx()
-  const isSilencesLoading = useSilencesIsLoading()
   const localSilences = useSilencesLocalItems()
+  const endpoint = useGlobalsApiEndpoint()
+  const {
+    setSilences,
+    setIsUpdating: setSilencesIsUpdating,
+    setError: setSilencesError,
+    setSilencesStatus,
+    setSilencesRegEx,
+  } = useSilencesActions()
+
+  const { data, error, isLoading } = useQuery({
+    queryKey: ["silences", endpoint], // Pass the query key as part of the object
```

Comment:
can be simplified when using predefined queries as described in the comment below within the alert worker file